proposals/lip-0049.md

```diff
@@ -457,7 +457,13 @@ The role of the channel terminated message is to inform chains that their chain
 
 ##### Parameters
 
-The `params` schema for the channel terminated CCM is the empty schema.
+```java
+channelTerminatedCCMParamsSchema = {
+    "type": "object",
+    "required": [],
+    "properties": {}
+}
+```
 
```

Comment:
Should we also define the verification then now to insure that the schema is obeyed?

Reply:
added